parsers/CL.py

```diff
@@ -126,7 +130,7 @@ def production_processor_historical(raw_data):
 
     ordered_data = sorted(combined.values(), key=itemgetter("datetime"))
 
-    if ENABLE_LIVE_PARSER:
+    if enable_live_parser:
         # For consistency with live API, hydro and geothermal must be squeezed into unknown
         for datapoint in ordered_data:
             if "unknown" not in datapoint:
```

Comment:
I believe this is already covered by the changes applied in the `_fetch_production` function, because that is the only place where the `production_processor_historical` function is called. Or perhaps I misunderstood?